src/poetry_plugin_bundle/plugin.py

```diff
@@ -19,7 +19,7 @@
 def load_command(name: str) -> Callable:
```

Comment:
This can now be simplified. See https://github.com/python-poetry/poetry-plugin-export/blob/04e7b2e92deafdc3498156c592714c1000ad2bc0/src/poetry_plugin_export/plugins.py#L16-L18

Reply:
very nice, fixed